src/implementor/simulation.py

```diff
@@ -203,6 +203,7 @@ def get_simulation_configuration(options, token):
         str(reference.train_prio_fault_configuration.id)
         for reference in simulation_configuration.train_prio_fault_configuration_references
     ]
+    run_ids = [str(run.id) for run in simulation_configuration.runs]
```

Comment:
Good question! The type is uuid. We'll use the function `to_dict()` to serialize it for the user as JSON in the response. Because JSON doesn't have uuid as a type, we have to cast it.